chips/sam4l/src/flashcalw.rs

```diff
@@ -317,6 +317,7 @@ impl FLASHCALW {
             FlashState::WriteUnlocking { page } => {
                 self.current_state.set(FlashState::WriteErasing { page: page });
                 self.flashcalw_erase_page(page);
+                DEFERED_CALL.set();
```

Comment:
Don't need this

Reply:
OK, removed it and I trust you, but I needed it before your changes on master, so should test this just in case before merging (which I can do first thing tomorrow)

Reply:
Tested. Works.